src/session.ts

```diff
@@ -175,20 +187,24 @@ class Session extends ServiceObject {
        */
       id: name,
       methods,
-      createMethod: (_, options, callback) => {
-        if (is.fn(options)) {
-          callback = options;
-          options = {};
-        }
-        return database.createSession(options, (err, session, apiResponse) => {
-          if (err) {
-            callback(err, null, apiResponse);
-            return;
-          }
+      createMethod: (  // tslint:disable-next-line no-any
+          _: any, optionsOrCallback: any|GetSessionCallback,
```

Comment:
- Instead of `any` for `_`, please just use `{}`
- For `optionsOrCallback`, please use a typed object

Reply:
Hi @JustinBeckwith , 
I used {} for optionsOrCallback.